code/modules/mob/new_player/new_player.dm

```diff
@@ -24,6 +24,8 @@
 /mob/new_player/Initialize()
 	. = ..()
 	verbs += /mob/proc/toggle_antag_pool
+	loc = null
```

Comment:
I know I left it alone previously, but possibly this should be `forceMove(null)` or even `ASSERT(loc == null)`.

Reply:
I considered that but wasn't sure if it was deliberately skipping Entered() etc being called.

Reply:
I don't know; they should never be spawned outside of the client/New anyway, but I guess you can bus them in (?).

Reply:
Removed these changes from this PR in any case.